app/src/main/java/com/elavon/converge/ElavonConvergeProcessorApplication.java

```diff
@@ -2,25 +2,10 @@
 
 import android.app.Application;
 
-import com.elavon.converge.core.TransactionManager;
-
+@Deprecated
 public class ElavonConvergeProcessorApplication extends Application {
```

Comment:
don't need this anymore. should remove?

Reply:
we might still need it for session management - so let's leave the basic class for now.